doc/book/connectors/__java.rst

```diff
@@ -2,4 +2,26 @@
                             Java
 =====================================================================
 
-See http://github.com/tarantool/tarantool-java/.
+There are two Java connectors available:
+
+*   `cartridge-java <http://github.com/tarantool/cartridge-java/>`__
+    supports both single Tarantool nodes and clusters,
+    as well as applications built using the :doc:`Cartridge framework </book/cartridge>` and its modules.
+    Tarantool team actively updates this module with newest Tarantool features.
```

Comment:
@akudiyar is this right? Do we intend to further develop this module?

Reply:
And I removed the line about the Netty framework because I don't see how it helps make the choice.

Reply:
```suggestion
    The Tarantool team actively updates this module with the newest Tarantool features.
```

Reply:
> is this right? Do we intend to further develop this module?
Yes, a good addition.

> I removed the line about the Netty framework
I think it must be left as it was, since in the Java world Netty framework is associated with smart and high load-ready things, and is a default option for an asynchronous network back-end. Also, it is opposed to a "hand-written" transport subsystem of the old driver.